### DISCUSSION TITLE
Classes in Citeseer dataset

### DISCUSSION THREAD

**Discussion by qubit0** [pyg-team/pytorch_geometric#6275] [Q&A]
There are 6 classes in `citeseer` dataset : Agents, Artificial Intelligence, Database, Human Computer Interaction, Machine Learning and Information Retrieval. In `PyG` those classes are represented from 0 to 5. How do these numbers map to the class's name?

**Comment by rusty1s** [comment]
I sadly don't know :( We use the pre-processed version from https://github.com/kimiyoung/planetoid. Maybe you have some luck asking there?
